modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointments.rb

```diff
@@ -230,6 +255,11 @@ def location(type, appointment_hash)
                 state: cc_location.dig(:address, :state),
                 zip_code: cc_location.dig(:address, :postal_code)
               }
+              if cc_location[:telecom].present?
+                phone_number = cc_location[:telecom].find { |contact| contact[:system] == CONTACT_TYPE[:phone] }[:value]
```

Comment:
How confident are you the find will return a hash? If it doesn't, this will error.

Reply:
the only examples in test data I saw was with the phone in the telecom hash but that's probably not a safe assumption. updated the logic so it doesn't error out. I used a `dig` at the end there so I could add in a safety operator. Not sure if there's a better way to do that since dig seems like it should only be used when there's multiple keys being called.